Gemfile.lock

```diff
@@ -862,4 +873,4 @@ RUBY VERSION
    ruby 2.6.3p62
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```

Comment:
any reason to change the bundler version? circleci does not runs with this

Reply:
Thanks! No there's no reason, I missed it by accident.